tekton/pipeline_run_test.go

```diff
@@ -106,6 +107,13 @@ var _ = Describe("PipelineRun", Ordered, func() {
 
 			Expect(newInternalRequestPipelineRun.Spec.Timeouts).To(Equal(timeouts))
 		})
+
+		It("should set the ServiceAccountName for the PipelineRun", func() {
```

Comment:
Maybe a test to ensure it's not added if not defined?

Reply:
added